gdal/swig/include/python/gdal_python.i

```diff
@@ -201,6 +201,52 @@ unsigned int wrapper_VSIFReadL( void **buf, unsigned int nMembSize, unsigned int
 %}
 %clear (void **buf );
 
+/* -------------------------------------------------------------------- */
+/*      VSIGetMemFileBuffer()                                           */
+/* -------------------------------------------------------------------- */
+
+%rename (VSIGetMemFileBuffer) wrapper_VSIGetMemFileBuffer;
```

Comment:
I'd like this function name to be suffixed by _unsafe since it is unsafe to call it  and close the file afterwards